pkg/antctl/raw/multicluster/join.go

```diff
@@ -0,0 +1,277 @@
+				   --clusterid=cluster-east
+
+# Join the ClusterSet from a member cluster with a config file. 
+  $ antctl mc join --config-file ./config.yaml
+
+# Config file example:
+apiVersion: multicluster.antrea.io/v1alpha1
+kind: ClusterSetConfig
+memberCluster:
+  namespace: kube-system
+  clusterID: cluster-east
+leaderCluster:
+  namespace: antrea-multicluster
+  secret: member-access-token
+  apiServer: https://172.18.0.3:6443
+  clusterID: cluster-north
+clusterSetID: clusterset
+`, "\n")
+
+func NewJoinCommand() *cobra.Command {
+	command := &cobra.Command{
+		Use:     "join",
+		Short:   "Join the ClusterSet from a member cluster",
+		Args:    cobra.MaximumNArgs(0),
+		Example: joinExamples,
+		RunE:    joinRunE,
+	}
+
+	o := JoinOptions{}
+	joinOpts = &o
+	command.Flags().StringVarP(&joinOpts.LeaderClusterOpt.Namespace, "leader-namespace", "", "", "Namespace of the leader cluster")
+	command.Flags().StringVarP(&joinOpts.LeaderClusterOpt.ClusterID, "leader-clusterid", "", "", "Cluster ID of the leader cluster")
+	command.Flags().StringVarP(&joinOpts.LeaderClusterOpt.Secret, "token-secret-name", "", "", "Name of the Secret resource that contains the leader cluster token")
+	command.Flags().StringVarP(&joinOpts.LeaderClusterOpt.APIServer, "leader-server", "", "", "API Server endpoint of the leader cluster")
+	command.Flags().StringVarP(&joinOpts.MemberClusterOpt.Namespace, "namespace", "n", "", "Namespace of the member cluster")
+	command.Flags().StringVarP(&joinOpts.MemberClusterOpt.ClusterID, "clusterid", "", "", "Cluster ID of the member cluster")
+	command.Flags().StringVarP(&joinOpts.ClusterSetID, "clusterset", "", "", "ClusterSet ID")
+	command.Flags().StringVarP(&joinOpts.ConfigFile, "config-file", "f", "", "Config file that defines all config options. If both command line options and config file are specified, "+
+		"the arguments in config file will be used.")
+
+	return command
+}
+
+func joinRunE(cmd *cobra.Command, args []string) error {
+	var err error
+	if err = joinOpts.validateAndComplete(); err != nil {
+		return err
+	}
+	k8sClient, err := common.NewClient(cmd)
+
+	memberClusterNamespace := joinOpts.MemberClusterOpt.Namespace
+	memberClusterID := joinOpts.MemberClusterOpt.ClusterID
+	memberClusterSet := joinOpts.ClusterSetID
+	createdRes := []map[string]interface{}{}
+	defer func() {
+		if err != nil {
+			fmt.Fprintf(cmd.OutOrStdout(), "Failed to join the ClusterSet. Deleting the created resources\n")
+			if err := common.Rollback(cmd, k8sClient, createdRes); err != nil {
+				fmt.Fprintf(cmd.OutOrStdout(), "Failed to rollback: %v\n", err)
+			}
+		}
+	}()
+
+	// The command prefers to use the Secret in the config, but if the Secret is not specified. It will create a Secret from the config.
+	if joinOpts.LeaderClusterOpt.Secret == "" {
+		if err := k8sClient.Create(context.TODO(), joinOpts.Secret); err != nil {
+			fmt.Fprintf(cmd.OutOrStdout(), "Failed to create the secret from the config file: %v\n", err)
```

Comment:
```suggestion
			fmt.Fprintf(cmd.OutOrStdout(), "Failed to create the Secret from the config file: %v\n", err)
```
please check other occurrences.

Reply:
Fixed.